.github/workflows/master.yml

```diff
@@ -40,7 +42,75 @@ jobs:
       - name: Test
         run: make test
         env:
-          TNB_DATABASES: '{"default":{"CLIENT":{"host":"mongo"}}}'
+          TNB_DATABASES: '{"default":{"CLIENT":{"host":"node-mongo"}}}'
           TNB_SECRET_KEY: dummy
           TNB_NODE_SIGNING_KEY: ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff
 # QUALITY-ASSURANCE-END
+
+  publish-image:
+    name: Publish docker images
+    needs: quality-assurance
+    runs-on: ubuntu-latest
+    steps:
+      - name: Login to GitHub Docker Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: $GITHUB_ACTOR
+          password: ${{ secrets.NODE_GITHUB_TOKEN }}
+      - name: Push node image to Github Packages
+        uses: docker/build-push-action@v2
+        with:
+          target: node
+          tags: ghcr.io/thenewboston-developers/node:latest
+          push: true
+          # TODO(dmu) LOW: Make a better way of forcing the download of latest blockchain state
+          #                https://thenewboston.atlassian.net/browse/BC-184
+          no-cache: true
+      - name: Push node-reverse-proxy image to Github Packages
```

Comment:
These two github packages are not covered with unittests, so they have to be built and pushed independently and before the unittest run (because unittests depend on node-mongo, also we need to build node-mongo in pr.yml, because of being a dependency).

Reply:
there should be something like:

```
jobs
  publish-node-reverse-proxy-image:
      - name: Push node-reverse-proxy image to Github Packages
        uses: docker/build-push-action@v2
        with:
          target: node-reverse-proxy
          tags: ghcr.io/thenewboston-developers/node-reverse-proxy:latest
          push: true
  publish-node-mongo-image:
      - name: Push node-mongo image to Github Packages
        uses: docker/build-push-action@v2
        with:
          target: node-mongo
          tags: ghcr.io/thenewboston-developers/node-mongo:latest
          push: true
```